src/components/bc-connector-list.ts

```diff
@@ -27,9 +29,26 @@ export class ConnectorList extends withTwind()(BitcoinConnectElement) {
       connectors.push(html`<bc-lnc-connector></bc-lnc-connector>`);
     }
 
-    return html` <div class="flex justify-center items-start flex-wrap gap-5">
-      ${connectors}
-    </div>`;
+    return html`
+      <div class="flex justify-center items-start flex-wrap gap-5">
+        ${connectors}
+      </div>
+      <div class="flex flex-col items-center w-full font-sans text-sm">
+        <h1
+          class="my-8 ${classes[
+            'text-neutral-primary'
+          ]} w-64 max-w-full text-center"
+        >
+          Don't have a bitcoin lightning wallet?
+          <a
+            class="no-underline font-bold ${classes['text-brand-mixed']} "
+            style="cursor: pointer;"
```

Comment:
instead of style here you can use `${classes.interactive}` as one of the classes. It also adds some other hover and click effects.